test/parallel/test-timers-active.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const active = require('timers').active;
+
+// active() should not create a timer for these
+var legitTimers = [
+  { _idleTimeout: 0 },
+  { _idleTimeout: 1 }
+];
+
+legitTimers.forEach(function(legit) {
+  active(legit);
+  // active() should mutate these objects
+  assert.notDeepEqual(Object.keys(legit), ['_idleTimeout']);
```

Comment:
Why not just `hasOwnProperty`? Also I feel that if we can check the values also it would be better.